src/Microsoft.TemplateEngine.Edge/Template/MatchInfo.cs

```diff
@@ -1,11 +1,19 @@
-﻿namespace Microsoft.TemplateEngine.Edge.Template
+namespace Microsoft.TemplateEngine.Edge.Template
 {
     public struct MatchInfo
```

Comment:
We should explicitly mark the layout as sequential here & verify that the additional fields don't break against clients that haven't been rebuilt